Are the tests working?

I cloned the project and executed the tests but it's not working. do you have any advice?
```
$ pytest -vv
```
output 
```
self = <tests.test_jsondiff.JsonDiffTests testMethod=test_a>

    def test_a(self):

        self.assertEqual({}, diff(1, 1))
        self.assertEqual({}, diff(True, True))
        self.assertEqual({}, diff('abc', 'abc'))
        self.assertEqual({}, diff([1, 2], [1, 2]))
        self.assertEqual({}, diff((1, 2), (1, 2)))
        self.assertEqual({}, diff({1, 2}, {1, 2}))
        self.assertEqual({}, diff({'a': 1, 'b': 2}, {'a': 1, 'b': 2}))
        self.assertEqual({}, diff([], []))
        self.assertEqual({}, diff(None, None))
        self.assertEqual({}, diff({}, {}))
        self.assertEqual({}, diff(set(), set()))

        self.assertEqual(2, diff(1, 2))
        self.assertEqual(False, diff(True, False))
        self.assertEqual('def', diff('abc', 'def'))
        self.assertEqual([3, 4], diff([1, 2], [3, 4]))
        self.assertEqual((3, 4), diff((1, 2), (3, 4)))
        self.assertEqual({3, 4}, diff({1, 2}, {3, 4}))
>       self.assertEqual({replace: {'c': 3, 'd': 4}}, diff({'a': 1, 'b': 2}, {'c': 3, 'd': 4}))
E       TypeError: unhashable type: 'Symbol'

tests/test_jsondiff.py:45: TypeError
```

--- a/.gitignore
+++ b/.gitignore
@@ -57,3 +57,5 @@ target/
 
 # PyCharm
 .idea
+
+.hypothesis/

--- a/jsondiff/symbols.py
+++ b/jsondiff/symbols.py
@@ -1,7 +1,11 @@
 
 class Symbol(object):
     def __init__(self, label):
-        self.label = label
+        self._label = label
+
+    @property
+    def label(self):
+        return self._label
 
     def __repr__(self):
         return self.label
@@ -11,6 +15,9 @@ class Symbol(object):
 
     def __eq__(self, other):
         return self.label == other.label
+    
+    def __hash__(self) -> int:
+        return hash(self.label)
 
 missing = Symbol('missing')
 identical = Symbol('identical')

